packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/configurationproperties/PortalDisabledTests.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.configurationproperties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import org.eclipse.sirius.components.core.api.IEditingContextSearchService;
+import org.eclipse.sirius.components.portals.description.PortalDescription;
+import org.eclipse.sirius.web.AbstractIntegrationTests;
+import org.eclipse.sirius.web.TestIdentifiers;
+import org.eclipse.sirius.web.application.editingcontext.EditingContext;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlConfig;
+
+/**
+ * Test the environment created when the support for portal is enabled.
+ *
+ * @author sbegaudeau
+ */
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, properties = { "sirius.web.enabled=*", "sirius.web.disabled=portal" })
+public class PortalDisabledTests extends AbstractIntegrationTests {
+
+    @Autowired
+    private IEditingContextSearchService editingContextSearchService;
+
+    @Test
+    @DisplayName("Given all Sirius Web features enabled, when representation descriptions are requested, then portal support is available")
```

Comment:
The display name does not match what is tested, i.e. portal not available when the feature is disabled.

Reply:
Indeed, I'll fix that